apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -31,7 +32,7 @@ export class InAppOutputRendererUsecase {
   }
 
   private buildRedirect(redirect?: InAppRedirectType) {
-    if (!(redirect && redirect.url && isValidURL(redirect.url))) {
+    if (!(redirect && redirect.url && isValidUrlForActionButton(redirect.url))) {
```

Comment:
dont we need to validate `redirect.target` it could be potentially undefined/null/non valid input.

Reply:
i'll add it